nutils/function.py

```diff
@@ -1951,7 +1951,7 @@ def dot(__a: IntoArray, __b: IntoArray, axes: Optional[Union[int, Sequence[int]]
     assert b.ndim == 1 and b.shape[0] == a.shape[0]
     b = _append_axes(b, a.shape[1:])
     axes = 0,
-  return sum(multiply(a, b), axes)
+  return sum(multiply(a, conjugate(b)), axes)
```

Comment:
leave dot as it is and add vdot for complex conjugate

Reply:
done